src/Files.App/Actions/FileSystem/OpenItemAction.cs

```diff
@@ -0,0 +1,38 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Helpers;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Files.App.Actions
+{
+	internal class OpenItemAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "Open".GetLocalizedResource();
+
+		public RichGlyph Glyph => new(opacityStyle: "ColorIconOpenFile");
+
+		public bool IsExecutable => context.HasSelection && context.SelectedItems.Count <= 10;
```

Comment:
What's the number 10? Should there be a constant variable for it?

Reply:
It was there in ContextFlyoutItemHelper, I assume it's more or less an arbitrary value to prevent the app from opening too many files at once

Reply:
If we still want to keep it, we should create a constant

Reply:
I'll do that